source/reverse-sync.txt

```diff
@@ -0,0 +1,189 @@
+.. _c2c-reverse-process:
+
+======================
+Reverse Sync Direction
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :class: singlecol
+   :depth: 1
+
+To reverse the direction of a sync operation, use the following
+procedure to commit your sync and call the ``reverse`` endpoint. 
+
+Before You Begin
+----------------
+
+Before you can reverse your sync direction, you must configure
+``mongosync`` and use the :ref:`c2c-api-start` endpoint to set the
+following parameters:
+
+- ``reversible`` to ``true``
+- ``enableUserWriteBlocking`` to ``true``.
+
+For more information on limitations and requirements of reversing sync,
+see :ref:`c2c-api-reverse`. 
+
+Steps
+-----
+
+Use these steps to reverse the direction of your sync:
+
+.. procedure::
+   :style: normal
+
+   .. step:: Verify the status of the ``mongosync`` process.
```

Comment:
[comment]

I think we can remove the styling backticks around mongosync in this title.

Reply:
Done!